modules/kubernetes-addons/agones/data.tf

```diff
@@ -1,5 +1,3 @@
-data "aws_region" "current" {}
-
 data "aws_security_group" "eks_worker_group" {
```

Comment:
can we audit if this is needed?

Reply:
used on `main.tf` for the security group rule resource `security_group_id = data.aws_security_group.eks_worker_group.id`

Reply:
is it used anywhere else? Other add-ons?

Reply:
Its used in few other add-ons as well.

Reply:
I couldn't find the same data source (`data "aws_security_group"`) being used on other addons where the id is:
```id = var.eks_worker_security_group_id```